akka-actor-typed-tests/src/test/scala/akka/actor/typed/scaladsl/ActorLoggingSpec.scala

```diff
@@ -183,98 +172,56 @@ class ActorLoggingSpec extends ScalaTestWithActorTestKit("""
           spawn(Behaviors.setup[String] { context =>
             context.log.debug("message")
             context.log.debug("{}", "arg1")
-            context.log.debug("{} {}", "arg1", "arg2")
-            context.log.debug("{} {} {}", "arg1", "arg2", "arg3")
-            context.log.debug("{} {} {} {}", "arg1", "arg2", "arg3", "arg4")
+            context.log.debug("{} {}", Array("arg1", "arg2"))
```

Comment:
If I look at the Slf4J logger it seems to have overloads for two placeholder values and then one with varargs so these shouldn't be needed?

Reply:
yes, I'll fix it.